### ISSUE TITLE
Request: Reload Keybind & Clip Remaining HUD Indication

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/U2Weapons#2]
- [ ] More than once having a near empty clip and only being able to auto reload has gotten me in trouble so it would be nice to have a reload keybind as well as a HUD clip count, recommend looking at Ballistic Weapons as all of those weapons use clips, have a visual HUD count and a reload keybind.

**Comment by GreatEmerald** [comment]
IIRC there already is a reload keybind.
Clip count would be useful but HUD changes are always tricky.

**Comment by HellDragon-HK** [comment]
I don't see one in the settings menu anywhere, how can I use it?
Ballistic weapons does it seemingly simply by having numbers above the ammo count to show clip and overall remaining ammo with text to identify which is which, would that really be that hard to do :S ?

**Comment by GreatEmerald** [comment]
Hard to do, no; but it would potentially conflict with LTBOMH. Needs to be investigated.

From the readme:
```
You can now reload manually. To do that, bind Reload to a key (in console, "set input key Reload").
```
Where `key` is the name of the key, I use `R`.

**Comment by HellDragon-HK** [comment]
Sorry I do read readme s but I must have missed that or forgotten it when actually needed.
Ballistic has a reload key so why doesn't it work with it?

**Comment by GreatEmerald** [comment]
Because I never used Ballistic. The keybind literally calls a function called "Reload"; I guess Ballistic calls its function something else.

**Comment by HellDragon-HK** [comment]
Nope, the same key works for both BW and U2, not Doom 3 though which has reload as alt fire, I have no idea why it wasn't working earlier but it does now.
